documentation/crds_users_guide/source/library_use.rst

```diff
@@ -181,7 +181,15 @@ crds.get_default_context()
 ..........................
 
 `get_default_context()` returns the name of the pipeline mapping which is 
-currently in operational use.
+currently the Latest Context
```

Comment:
"which is currently the Latest Context if observatory is HST or Roman; for JWST this is the Build Context."